### ISSUE TITLE
Move to new Go 1.16 embed tooling for bundling the admin

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#844]
Right now we use a 3rd party tool, `pkged` to bundle the admin into the Go codebase. With 1.16 Go supports native tooling for this.

https://pkg.go.dev/embed

See https://github.com/owncast/owncast/blob/develop/build/admin/bundleAdmin.sh#L37

**Comment by gabek** [comment]
I looked into this and it doesn't exactly work the way that would translate to how we're doing things now.

With `go:embed` it only embeds when building.  So when running from source you'd need a built version of the admin on the filesystem.  While not the end of the world, it would require everyone who works on Owncast core to know how to acquire, install the dependencies, and build the admin.  The alternative is we keep a built copy the Admin javascript in the Owncast core project (instead of how we keep it in pkged.go right now).

So in short, there's no way to pre-embed and allow people developing Owncast to use it.  So for now I'm going to leave it how it is because it works fine.  But I'd love to find a way to use the new, supported features in Go.
